Frends.FTP.ListFiles/Frends.FTP.ListFiles/ListFiles.cs

```diff
@@ -20,22 +20,23 @@ public class FTP
         /// </summary>
         /// <param name="input">Input parameters</param>
         /// <param name="cancellationToken">Token to stop task. This is generated by Frends.</param>
-        /// <returns>List { DateTime Modified, string Name, string FullPath, long Size, bool Success }</returns>
+        /// <returns>List { ListObject }</returns>
```

Comment:
Can you convert it back to previous way that has the actual properties of ListObject?